From 7ccc2b76888443380f516f0ae34ce0d8b38194ac Mon Sep 17 00:00:00 2001
From: Mo Zhou <cdluminate@gmail.com>
Date: Wed, 1 Apr 2020 17:28:52 +0800
Subject: [PATCH] CMake: Bump SOVERSION to "1" due to the breaking changes in
 API/ABI. (fixes #661)

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a99eb0cf..e0f7f2f2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,7 +29,7 @@ set(PROJECT_VERSION_MAJOR 0)
 set(PROJECT_VERSION_MINOR 3)
 set(PROJECT_VERSION_PATCH 0)
 set(SIMDJSON_LIB_VERSION "0.3.0" CACHE STRING "simdjson library version")
-set(SIMDJSON_LIB_SOVERSION "0" CACHE STRING "simdjson library soversion")
+set(SIMDJSON_LIB_SOVERSION "1" CACHE STRING "simdjson library soversion")
 
 option(SIMDJSON_IMPLEMENTATION_HASWELL "Include the haswell implementation" ON)
 option(SIMDJSON_IMPLEMENTATION_WESTMERE "Include the westmere implementation" ON)
