https://github.com/lemire/simdjson/pull/98
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7ded8f6..ed90934 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,7 +10,7 @@ endif()
 project(simdjson)
 set(SIMDJSON_LIB_NAME simdjson)
 set(SIMDJSON_LIB_VERSION "0.0.1" CACHE STRING "simdjson library version")
-set(SIMDJSON_LIB_SOVERSION "0.0.1" CACHE STRING "simdjson library soversion")
+set(SIMDJSON_LIB_SOVERSION "0" CACHE STRING "simdjson library soversion")
 
 if(NOT MSVC)
 option(SIMDJSON_BUILD_STATIC "Build a static library" OFF) # turning it on disables the production of a dynamic library
